### ISSUE TITLE
RCBugFix won't compile - planner.cpp syntax issue

### ISSUE THREAD

**Issue by Bob-the-Kuhn** [MarlinFirmware/Marlin#5039]
The latest RCBugFix won't compile because of a syntax issue in planner.cpp

Lines 1023-1026 now read

```
    #define LIMIT_ACCEL(AXIS) do{ \
      if (max_acceleration_steps_per_s2[AXIS] < (accel * block->steps[AXIS]) / block->step_event_count) 
        accel = (max_acceleration_steps_per_s2[AXIS] * block->step_event_count) / block->steps[AXIS]; 
    }while(0)

```

1024 & 1025 need a "\" at the end.  They should read

```
    #define LIMIT_ACCEL(AXIS) do{ \
      if (max_acceleration_steps_per_s2[AXIS] < (accel * block->steps[AXIS]) / block->step_event_count) \
        accel = (max_acceleration_steps_per_s2[AXIS] * block->step_event_count) / block->steps[AXIS]; \
    }while(0)
```

**Comment by Blue-Marlin** [comment]
Fix for the commit dc2cb84 (Revert acceleration limiting to avoid overflow) #5037

**Comment by Bob-the-Kuhn** [comment]
Great
